### PR TITLE
WIZZIE-2212 Add missing http2k module config file

### PR DESCRIPTION
**Pull request by Chorro**
Youtrack: close #WIZZIE-2212

### CODE REVIEW

File: cli/include/config.bash [hunk 1]
```diff
@@ -421,6 +421,9 @@ zz_enable_disable_modules() {
             ;;
         esac
     done
+
+    # Remove unused/orphans containers
+    "${PREFIX}/bin/prozzie" up --remove-orphans -d
```
Comment (eugpermar) [inline cli/include/config.bash]: Could this be in a separate commit?

**Comment by eugpermar** [comment]
Hi! Could http2k have it own enabling/disabling test?

File: cli/include/config.bash (hunk 1 above)
Comment (Chorro) [reply cli/include/config.bash]: Done 👍

**Comment by Chorro** [comment]
I've added http2k tests in [testEnableModule](https://github.com/wizzie-io/prozzie/pull/52/files#diff-077fb1ccf4e716f238eaf2ca49d08414R323) and [testDisableModule](https://github.com/wizzie-io/prozzie/pull/52/files#diff-077fb1ccf4e716f238eaf2ca49d08414R339) tests.

**Comment by eugpermar** [comment]
Could you check that an http message can be sent with curl when the module is enabled, and that is not possible to send http message when the module is disabled?

**Comment by Chorro** [comment]
Ok. I've added the checks in [testEnableModule](https://github.com/wizzie-io/prozzie/pull/52/files#diff-077fb1ccf4e716f238eaf2ca49d08414R340) and [testDisableModule](https://github.com/wizzie-io/prozzie/pull/52/files#diff-077fb1ccf4e716f238eaf2ca49d08414R364) tests.
